components/device.js

```diff
@@ -0,0 +1,112 @@
+const StdLib = require('@doctormckay/stdlib');
+const Helpers = require('./helpers.js');
+const SteamUser = require('../index.js');
+
+/**
+ * Get authorized devices.
+ * @param {boolean} [inclTokens=false] - If true, include canceled devices
```

Comment:
`inclTokens`? Also, while `canceled` isn't technically incorrect, usually `cancelled` is before.

Reply:
oh ... sorry hehe. Anyway, I fixed the "inclTokens" and changed the "canceled".